Source/Core/DolphinWX/CMakeLists.txt

```diff
@@ -101,7 +101,6 @@ endif()
 
 set(DOLPHIN_EXE ${DOLPHIN_EXE_BASE})
 
-include(FindGettext)
```

Comment:
It is my understanding we need gettext on all platforms, so I added it to the main CMakeLists.txt file and removed it from DolphinWX/CMakeLists.txt